lib/index.js

```diff
@@ -17,7 +21,10 @@ module.exports = function (options) {
     tree.match([{ tag: 'img' }, { tag: 'amp-img' }], function (imgNode) {
       if (imgNode.skip) return imgNode
       var classes = imgNode.attrs && imgNode.attrs.class && (imgNode.attrs.class.split(' ') || [])
-      var isIgnore = options.classIgnore.filter(className => classes.includes(className)).length > 0
+      var extension = imgNode.attrs.src.split('.').pop()
+      var isIgnore =
+        (options.classIgnore.filter(className => classes.includes(className)).length > 0) ||
+        (options.extensionIgnore.filter(fileExtension => fileExtension === extension).length > 0)
```

Comment:
slightly heavy construction can in a separate method/s ?

Reply:
Sure. You mean something like this?

```javascript
var isIgnoredByClass = options.classIgnore.filter(className => classes.includes(className)).length > 0
var isIgnoredByExtension = options.extensionIgnore.filter(fileExtension => fileExtension === extension).length > 0
var isIgnore = isIgnoredByClass || isIgnoredByExtension
```

Reply:
yes, that's better already

Reply:
Done